src/module.ts

```diff
@@ -180,8 +185,9 @@ export default defineNuxtModule<NuxtI18nOptions>({
     addPlugin(resolve(runtimeDir, 'plugins/i18n'))
     addPlugin(resolve(runtimeDir, 'plugins/switch-locale-path-ssr'))
 
+    const resolver = createResolver(import.meta.url)
     // for composables
-    nuxt.options.alias['#i18n'] = resolve(distDir, 'runtime/composables/index.js')
+    nuxt.options.alias['#i18n'] = resolver.resolve('./runtime/composables/index')
```

Comment:
add `.js` extension here

Reply:
Looks like adding the extension breaks explicit imports (when having auto imports disabled) https://github.com/nuxt-modules/i18n/actions/runs/10947564325/job/30397002095?pr=3112